seclytics-augur-pxdr/playbooks/playbooks.json

```diff
@@ -4,245 +4,200 @@
     {
       "@context": "/api/3/contexts/WorkflowCollection",
       "@type": "WorkflowCollection",
-      "name": "Sample - Seclytics Augur - 1.0.1",
-      "description": "Sample playbooks for \"Seclytics Augur\" connector. If you are planning to use any of the sample playbooks in your environment, ensure that you clone those playbooks and move them to a different collection, since the sample playbook collection gets deleted during connector upgrade and delete.",
+      "name": "Sample - Seclytics Augur pXDR - 1.0.0",
+      "description": "Sample playbooks for \"Seclytics Augur pXDR\" connector. If you are planning to use any of the sample playbooks in your environment, ensure that you clone those playbooks and move them to a different collection, since the sample playbook collection gets deleted during connector upgrade and delete.",
       "visible": true,
-      "image": "/api/3/images/e042ab97-758e-4901-8b18-6c9a0a068cc7",
-      "id": 22,
-      "createDate": 1632447929,
-      "modifyDate": 1632460063,
-      "recordTags": [
-        "SeclyticsAugur"
-      ],
+      "image": "/api/3/images/1f492f37-1e69-461d-add4-1e6c1667eea7",
+      "id": 1150,
+      "createDate": 1645616293,
+      "modifyDate": 1645616293,
+      "recordTags": [],
       "workflows": [
         {
           "@type": "Workflow",
           "triggerLimit": null,
-          "name": "Get Domain Reputation",
+          "name": "Get File Reputation",
           "aliasName": null,
-          "tag": null,
-          "description": "Get reputation of a domain from Seclytics.",
+          "tag": "#Seclytics Augur pXDR",
+          "description": "Retrieves a report for the file that you have submitted using file hash.",
           "isActive": false,
           "singleRecordExecution": false,
           "remoteExecutableFlag": false,
           "parameters": [],
           "synchronous": false,
-          "lastModifyDate": 1632459866,
-          "collection": "/api/3/workflow_collections/d2162198-cc58-4c35-9761-5f5366696bd2",
+          "lastModifyDate": 1645616586,
+          "collection": "/api/3/workflow_collections/cbb6c538-09a2-4cc7-abc0-e0f717b7cf50",
           "versions": [],
-          "triggerStep": "/api/3/workflow_steps/12ec6a77-6f37-4822-9c8f-0cb043380de6",
+          "triggerStep": "/api/3/workflow_steps/4c19514b-6435-4efb-ba2d-ccfd9f21753c",
           "steps": [
             {
               "@type": "WorkflowStep",
-              "name": "Start",
+              "name": "Alerts",
```

Comment:
why this name?

Reply:
The Playbooks are generated by our automation script. So not sure why this is given as Alerts.